src/app/comment-period/comment-period.component.html

```diff
@@ -189,7 +189,7 @@ <h4 class="card-header">Open House Dates</h4>
       <div class="modal-content">
         <div class="modal-header pb-0">
           <h3 class="modal-title comment-modal-title">Related Documents <span class="comment-modal-title__large">{{ commentPeriod.informationLabel }}</span></h3>
-          <button class="btn btn-default" (click)="hideAllSteps()" data-dismiss="modal" aria-label="Close">
+          <button class="btn btn-default" data-dismiss="modal" aria-label="Close">
```

Comment:
Why is ```hideAllSteps()``` missing from the review? Did the function exist already and was not being called (here)?

Reply:
Nevermind... -.-

Reply:
It was copy/pasted from another control. It does not need to be here.